1908-01-20.xml

```diff
@@ -0,0 +1,5143 @@
+                        </row>
+                        <row>
+                            <cell>Stock</cell>
+                            <cell><measure unit="balles">260,000</measure></cell>
+                            <cell><measure unit="balles">661,000</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Flottant</cell>
+                            <cell><measure unit="balles">12,000</measure></cell>
+                            <cell><measure unit="balles">153,000</measure></cell>
+                        </row>
+                        <row role="label">
+                            <cell/>
+                            <cell cols="2">EGYPTIEN</cell>
+                        </row>
+                        <row role="label">
+                            <cell/>
+                            <cell>1906</cell>
+                            <cell>1905</cell>
+                        </row>
+                        <row role="label">
+                            <cell/>
+                            <cell>Balles</cell>
+                            <cell>Balles</cell>
+                        </row>
+                        <row>
+                            <cell>Ventes Semaine</cell>
+                            <cell><measure unit="balles">1,500</measure></cell>
+                            <cell><measure unit="balles">2,400</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Forwarded</cell>
+                            <cell><measure unit="balles">2,000</measure></cell>
+                            <cell><measure unit="balles">1,800</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Importation</cell>
+                            <cell><measure unit="balles">150</measure></cell>
+                            <cell><measure unit="balles">5,000</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Exportation</cell>
+                            <cell><measure unit="balles">300</measure></cell>
+                            <cell><measure unit="balles">1,100</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Stock</cell>
+                            <cell><measure unit="balles">19,000</measure></cell>
+                            <cell><measure unit="balles">33,000</measure></cell>
+                        </row>
+                        <row>
+                            <cell>Flottant</cell>
+                            <cell><measure unit="balles">7,000</measure></cell>
+                            <cell><measure unit="balles">7,000</measure></cell>
+                        </row>
+                    </table>
+                </div>
+                
+                
+            </div>
+            <pb n="7"/>
```

Comment:
this whole page is available as a template: https://dig-eg-gaz.github.io/issues/templates/#1907-08-variant